python/requirements/ml/requirements_compat.txt

```diff
@@ -0,0 +1,16 @@
+# ML libraries
```

Comment:
Can we document what compatibility means here (e.g. Python 3.6) so that in the future people know what should or should not be updated in this file?